src/core/providers/arcgis/qgsarcgisrestquery.cpp

```diff
@@ -88,11 +88,27 @@ QgsRectangle QgsArcGisRestQueryUtils::getExtent( const QString &layerurl, const
   }
 
   QgsRectangle rect;
-  const QVariantMap coords = res.value( QStringLiteral( "extent" ) ).toMap();
-  rect.setXMinimum( coords.value( QStringLiteral( "xmin" ) ).toDouble() );
-  rect.setYMinimum( coords.value( QStringLiteral( "ymin" ) ).toDouble() );
-  rect.setXMaximum( coords.value( QStringLiteral( "xmax" ) ).toDouble() );
-  rect.setYMaximum( coords.value( QStringLiteral( "ymax" ) ).toDouble() );
+  do
```

Comment:
Could you move this to a function in QgsArcGisRestUtils? It'll be cleaner that way

Reply:
See how you like 0b670acd09d5d8286fdebb69dd649672be3dad10